.prout.json

```diff
@@ -0,0 +1,6 @@
+{
+  "checkpoints": {
+    "Checker": { "url": "https://checker.typerighter.gutools.co.uk/healthcheck", "overdue": "15M" },
+    "Rule Manager": { "url": "https://manager.typerighter.gutools.co.uk/healthcheck", "overdue": "15M" }
```

Comment:
These urls both have the advantage of not requiring authentication, so that Prout can reach them easily to get the git commit id.